lm75bd/lm75bd.c

```diff
@@ -26,8 +26,14 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
-  
```

Comment:
check if temp is a null pointer (always check if pointer args are null at the start of a function)